dyn_em/module_initialize_real.F

```diff
@@ -3089,7 +3104,17 @@ SUBROUTINE init_domain_rk ( grid &
 
       !  Split NUDAPT  Urban Parameters
 
-      IF ( ( config_flags%sf_urban_physics == 1 ) .OR. ( config_flags%sf_urban_physics == 2 ) .OR. ( config_flags%sf_urban_physics == 3 ) ) THEN
+      distributed_aerodynamics_if: IF (config_flags%sf_urban_physics == 1 .AND. config_flags%use_distributed_aerodynamics) THEN
+         DO j = jts , MIN(jde-1,jte)
+            DO i = its , MIN(ide-1,ite)
+              IF (grid%ivgtyp(i, j) == model_config_rec%isurban(grid%id)) THEN
+                grid%frc_urb2d(i, j) = MAX(0.1, MIN(0.9, 1 - grid%shdavg(i, j) / 100.))
+              END IF
```

Comment:
urban fraction set to annual mean vegetation fraction?

Reply:
I asked the same question to the developer and he said this treatment seems most reasonable for their new scheme.

Reply:
I would be concerned about urban areas in dry/desert regions. Why not trust frc_urb2d more?

Reply:
@dudhia
* Is FRC_URB2D available globally? I am sorry for my lack of understanding but I've never used that static data before.
* May be we can add a switch to allow users to pick between our way of estimating FRC_URB2D and the static FRC_URB2D input file.

Reply:
we can think of it later